### ISSUE TITLE
Emails récapitulatifs journalier

### ISSUE THREAD

**Issue by yannski** [betagouv/civilsdeladefense#238]
Type de notification | Contenu | Grand employeur | Employeur | ES | BRH | BANT | Remarques
-- | -- | -- | -- | -- | -- | -- | --
Publication d’une offre | Entité et référence de l’offre | X | X  | X | X | X |  emails à tous les acteurs sauf l'auteur
Nouveau candidat |   |   | X |   |   |   |  
Candidat retenu |   | X |   | X | X | X |  
Contrat reçu |   | X |   | X | X | X |  
Candidat affecté |   | X |   | X | X | X |  
Avertissement PJ complet contrat |   |   |   |   |   | X | Rien jusqu’à retenu
Avertissement PJ complet dossier prise en charge |   |   |   |   | X |   |

**Comment by Grainedepaillette** [comment]
Destinataire : BANT 

Ajouter 3 types de notifications supplémentaires : 
- lorsque passage de l'offre de l'état brouillon à publier 
- lorsque passage de l'offre de l'état publiée à mise en pause 
- lorsque passage de l'offre de l'état publiée à archivée

**Comment by Grainedepaillette** [comment]
@btrd Voici les specs attendues : 

Le récapitulatif journalier doit faire apparaître les rubriques suivantes : 

1. Nouvelles offres (déjà développé)
2. Offres publiées récemment (déjà développé) --> MAIS A RETIRER car information non nécessaire si les autres catégories sont disponibles 
3. Offres passées de l'état Brouillon à l'état Publié 
4. Offres passées de l'état Publié à l'état Mis en pause 
5. Offres passées de l'état Publié à l'état Archivé
6. Nouvelles candidatures 
7. Candidatures retenues 
8. Candidatures retenues pour lesquelles le dossier est complet et valide (PJ téléversées et validées) pour élaboration du contrat 
--> Correspond aux offres pour lesquelles une candidature se situe dans la colonne "rédaction contrat" 
9. Candidatures pour lesquelles le contrat est signé 
--> Correspond aux offres pour lesquelles une candidature se situe dans la colonne "contrat reçu" 

Pour chaque rubrique, afficher : 
- La référence de l'offre 
- L'intitulé de l'offre 
- Le lien URL de l'offre 

L'affichage de ces données pour chacune des rubriques ci - dessus est conditionné par le changement d'état pour les offres et les candidatures les dernières 24H (entre 2 récapitulatifs journaliers).

**Comment by Grainedepaillette** [comment]
> Candidatures retenues
Candidatures retenues pour lesquelles le dossier est complet et valide (PJ téléversées et validées) pour élaboration du contrat
--> Correspond aux offres pour lesquelles une candidature se situe dans la colonne "rédaction contrat"
Candidatures pour lesquelles le contrat est signé
--> Correspond aux offres pour lesquelles une candidature se situe dans la colonne "contrat reçu"

@btrd Pour ces rubriques ajouter le référence et l'intitulé de l'offre + le nom et prénom du candidat + URL de la candidature sur cette offre

**Comment by btrd** [comment]
Disponible ici https://erecrutement-cvd-staging-pr1089.osc-fr1.scalingo.io

J'ai ajouté une page pour voir le rapport journalier qui sera envoyé le lendemain matin, ça permet de simplifier les tests de votre côté

https://erecrutement-cvd-staging-pr1089.osc-fr1.scalingo.io/admin/account/notifications

**Comment by btrd** [comment]
@mch2608 J'ai oublié, c'est quoi la règle concernant les BANT ? Ils reçoivent les notifications pour toutes les offres ?

**Comment by mch2608** [comment]
> @mch2608 J'ai oublié, c'est quoi la règle concernant les BANT ? Ils reçoivent les notifications pour toutes les offres ?

Oui, le responsable fonctionnel reçoit bien le récapitulatif journalier avec l'ensemble des offres

**Comment by btrd** [comment]
C'est mis en place

**Comment by btrd** [comment]
@mch2608 J'ai corrigé un bug sur les notifications concernant une candidature, les prochaines notifications seront bien associés

**Comment by mch2608** [comment]
Merci. 
Après test, si je reprends les points de @Grainedepaillette. 
3. Opérationnel 
4. Modifier l'intitulé de la rubrique pour "Offres passées de l'état Publié à l'état Mis en pause"
5. Modifier l'intitulé de la rubrique pour "Offres passées de l'état Publié à l'état Archivé" 
6. 7. 8. 9. Opérationnel mais fonctionne seulement avec les nouvelles offres. Je me demande donc si la fonctionnalité sera opérationnelle pour les offres déjà existantes sur la plateforme. 

PS : j'avais écris le commentaire à 16h00 mais oublié de cliquer sur "Comment". Entre temps, je voulais retester pour être sur et je me suis d'un bug : lorsqu'on clique sur l'onglet Recap journalier, on est renvoyé vers un message d'erreur en front office. 


https://user-images.githubusercontent.com/58481185/127545627-c3a1a85a-8ef0-4e95-9f6d-ba3e4f8df8bd.mov

**Comment by btrd** [comment]
>  6. 7. 8. 9. Opérationnel mais fonctionne seulement avec les nouvelles offres. Je me demande donc si la fonctionnalité sera opérationnelle pour les offres déjà existantes sur la plateforme.

Qu'est-ce qui ne fonctionne pas avec les offres déjà existantes ?

PS: C'est corrigé

**Comment by mch2608** [comment]
Non c'est ok, finalement ! une erreur de ma part.
